moveit_core/collision_detection_fcl/CMakeLists.txt

```diff
@@ -50,23 +50,23 @@ if(BUILD_TESTING)
     set(append_library_dirs "${CMAKE_CURRENT_BINARY_DIR};${CMAKE_CURRENT_BINARY_DIR}/../collision_detection;${CMAKE_CURRENT_BINARY_DIR}/../robot_state;${CMAKE_CURRENT_BINARY_DIR}/../robot_model;${CMAKE_CURRENT_BINARY_DIR}/../utils")
   endif()
 
-  ament_add_gtest(test_fcl_collision_detection test/test_fcl_collision_detection.cpp
-    APPEND_LIBRARY_DIRS "${append_library_dirs}")
+  # ament_add_gtest(test_fcl_collision_detection test/test_fcl_collision_detection.cpp
+  #   APPEND_LIBRARY_DIRS "${append_library_dirs}")
```

Comment:
Why are you disabling FCL tests??

Reply:
You almost made me doubt myself ;) I stumbled over this when resolving conflicts and meant to remove it. The test was renamed some time ago https://github.com/ros-planning/moveit/pull/1856/files#diff-8c0f112273f9ce6b5fbf63828202c055R30 and is still enabled a couple lines below. The commented part was the attempted fixup during the alpha migration and can safely be removed. No test coverage lost.

Reply:
removed the comment